src/form-fields/DropDown.component.js

```diff
@@ -59,9 +61,20 @@ DropDown.propTypes = {
     ]),
     includeEmpty: PropTypes.bool,
     emptyLabel: PropTypes.string,
-    noOptionsLabel: PropTypes.string,
 };
+
 DropDown.defaultProps = {
+    value: PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.number,
+        PropTypes.bool,
+    ]),
+    menuItems: PropTypes.oneOfType([
+        PropTypes.array,
+        PropTypes.object,
+    ]),
+    onFocus: PropTypes.func,
+    onBlur: PropTypes.func,
     includeEmpty: false,
```

Comment:
If you add a default value for `noOptionsLabel` here, the code in `render()` can be simplified as well since it will be safe to assume that it will always have a value.